### PR TITLE
Feature: APP-601 empty state web app

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

This PR introduces the `EmptyState` component from the `ui-components` library to the web-app. The components are added to the dashboard, where they are used when the proposal or the transfers snapshot is empty.

In order to do this, the `EmptyState` component, as well as the `IlluHuman` component it makes use of, have had to be heavily refactored. The issues were:
1. Imports using `require` do not work natively in the browser. The component has therefore been refactored to use the standard ESM imports.
2. The relative and absolute positioning as well as the fixed width/height on various `div` component were making it impossible to bound the nodes containing text (title and description) to their parent. This would lead to a text overflow.

Task: [601](https://aragonassociation.atlassian.net/browse/APP-601?atlOrigin=eyJpIjoiYjJlODI4ZjEyOWQ2NDNhOWExODNhNmQ3ZTNkYjkzYjEiLCJwIjoiaiJ9)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/ui-components/src/components/stateEmpty/stateEmpty.tsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
   title: string;
   description?: string;
-  buttonLabelPrimary: string;
-  buttonLabelSecondary?: string;
+  primaryButton: ButtonProps;
+  secondaryButton?: ButtonProps;
+  renderHtml?: boolean;
 };
 
 export const StateEmpty: React.FC<StateEmptyProps> = ({
```
Comment (sepehr2github) [inline packages/ui-components/src/components/stateEmpty/stateEmpty.tsx:19]: I faced ```Cannot read properties of undefined (reading 'label')``` when I wanted to check stateEmpty on storybook

**Review by sepehr2github — COMMENTED** [review]
Wow, Vali, It was a vast refactor! Thank you for your hard work ;D. Just left a minor comment.

File: packages/ui-components/src/components/stateEmpty/stateEmpty.tsx (hunk 1 above)
Comment (WissenIstNacht) [reply packages/ui-components/src/components/stateEmpty/stateEmpty.tsx:19]: Thanks, completely forgot to adapt that story!

**Review by Fabricevladimir — APPROVED** [review]
Thanks for the refactor

**Review by sepehr2github — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**